src/app/models/book-model.ts

```diff
@@ -0,0 +1,28 @@
+import { bookType } from '../book-type.enum';
+import { BookBorrowData } from './book-borrow-data-model';
+
+export class Book {
+  public id: number;
+  public nameBook: string;
+  public bookCategory: bookType;
+  public author: string;
+  public price: number;
```

Comment:
Good practice is to close price in separate object: It would be something like: Price: {value: xxx, currency: xxx}. You can do new class for that if you want to.